website/src/jest/css/jest.css

```diff
@@ -1675,3 +1663,94 @@ div.video-shoutout {
 .miniShowcaseSection .button {
   text-decoration: none !important;
 }
+
+/** Footer **/
+footer.nav-footer {
+  box-sizing: border-box;
+  border: none;
+  font-weight: 300;
```

Comment:
Footer should look much better with `font-weight: 400`, 300 is hardly readable here